packages/cozy-sharing/src/components/ShareModal/EditableSharingModal.jsx

```diff
@@ -47,8 +31,6 @@ export const EditableSharingModal = ({ document, ...rest }) => {
 
   return (
     <DumbShareModal
```

Comment:
nit: As you moved the queries inside of child component, I suppose we cannot call this a `dumb` component anymore 😅